docs/modules/ROOT/pages/references/functional-requirements/portal.adoc

```diff
@@ -96,3 +96,16 @@ A lot of sanity checks are needed:
 * Create resources
 * Update resources
 * Delete resources
+
+== Feature: Manage Billing Information
+
+> As an xref:references/glossary.adoc#_organization-owner[Organization Owner] +
+> I want to choose how my resources are billed +
+> So that I can have I can simplify the setup for me and my users.
```

Comment:
While we should look at things from a users' perspective, here it does not work. A user does not have a motivation to give us that information. It is actually us that have a need here.

```suggestion
> As a service provider +
> I want users to link Orgnaisations to BillingEntities +
> So that I know where the invoice can be sent to.
```

Or something along that line. Feel free to improve.